docs/dataset-therapeutics.md

```diff
@@ -0,0 +1,34 @@
+The Therapeutics dataset contains information on ...
+
+!!! warning
+    Therapeutics data can only be used ...
```

Comment:
@amirmehrkar can you remind me who this is restricted to please?

Reply:
@amirmehrkar has confirmed it is just subject to the usual restrictions, nothing additional